.github/workflows/node_deploy.yml

```diff
@@ -0,0 +1,58 @@
+name: AWS DEPLOY
+# turn off deploy to AWS
+
+on:
+  pull_request:
+    types: [closed]
+    branches:
+      - main
+
+jobs:
+  aws-deploy:
+    runs-on: ubuntu-latest
+    env:
+      DB_HOST: ${{ secrets.AWS_DB_HOST }}
+      DB_PW: ${{ secrets.AWS_DB_PW}}
+      DB_USER: ${{ secrets.AWS_DB_USER }}
+      DB_NAME: ${{ secrets.AWS_DB_NAME }}
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: dev
```

Comment:
seems like a mistake to checkout `dev` on a pull request merged to `main` in order to do the build